tests/core/pm-module/test_pm_init.py

```diff
@@ -1,8 +1,7 @@
 import pytest
 
-from web3.pm import (
-    PM,
-)
+# Skip all test that depend on `ethpm` until dependency is stabilized.
+pytest.importorskip("ethpm")
```

Comment:
Our general preference is to fully remove and then re-add later when ready in cases like this.  Git makes digging this stuff from history pretty easy.

Reply:
Sounds good

Reply:
Looks like @carver has something else in mind: https://github.com/ethereum/web3.py/pull/893/commits/7a0fb00e5ab47dbfdde146329754b51bfe969667

Reply:
@voith Good to know - If it doesn't make a big difference, I'm leaning towards removing the tests from master, and moving `ethpm` integration to it's own branch on `web3` until we reach a stable checkpoint. @davesque ?